desktop/core/src/desktop/templates/common_share.mako

```diff
@@ -21,7 +21,7 @@ from django.utils.translation import ugettext as _
   <a>
     <div>
       <i data-bind="css: icon"></i>
-      <span data-bind="html: label"></span>
+      <span data-bind="text: label"></span>
```

Comment:
The html binding should already be sanitized, see [ko.html.js](https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/ko/bindings/ko.html.js)

Reply:
+1, maybe you were on an old Hue (can check the history of ko.html.js)

Reply:
Hi @JohanAhlen ,

its true that already the xss attack is been taken care at ko.html.js  , But ideally the label should be displayed as text .

you can see further implementations of how users are handled as as Text : https://github.com/cloudera/hue/blob/dc67dc6ad4df0a2db0f16ef9216fb4bf87045e67/desktop/core/src/desktop/templates/common_share.mako#L48

https://github.com/cloudera/hue/blob/dc67dc6ad4df0a2db0f16ef9216fb4bf87045e67/desktop/core/src/desktop/templates/common_share.mako#L58